templates/app_client/lib/app_client.rb.erb

```diff
@@ -55,7 +55,8 @@ module <%= gem_name_pascal %>
     end
 
     def adapter_factory
-      @adapter_factory ||= <%= gem_name_pascal %>.const_get("#{api_version.upcase}")::Adapters::Http
+      version = options[:api_version] ? options[:api_version] : api_version
```

Comment:
can't we just use `api_version` because the `||` is in that method on line 49 above?

Reply:
oooo good catch!